files/zh-cn/glossary/index.md

```diff
@@ -14,7 +14,7 @@ Web 技术文档和代码中含有大量的术语和缩写。本术语表给出
 
 ## 为本表作贡献
 
-> **备注：** 编辑术语表是一项永无止境的工作，你可以[添加新的条目](/zh-CN/docs/MDN/Contribute/Howto/Write_a_new_entry_in_the_Glossary)或改进、完善现有条目。
+> **备注：** 编辑术语表是一项永无止境的工作，你可以 [添加新的条目](/zh-CN/docs/MDN/Contribute/Howto/Write_a_new_entry_in_the_Glossary) 或改进、完善现有条目。
```

Comment:
We just do not add space between Chinese (even if it's a link), see: <https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E4%B8%AD%E6%96%87%E5%92%8C%E6%8B%89%E4%B8%81%E8%AF%AD%E7%B3%BB%E6%96%87%E5%AD%97%E9%97%B4%E5%8A%A0%E7%A9%BA%E6%A0%BC>

Reply:
I will fix this rule.